tools/binlog/tidb-binlog-kafka.md

```diff
@@ -0,0 +1,421 @@
+---
+title: TiDB-Binlog kafka 部署方案
+category: tools
+---
+
+# TiDB-Binlog Kafka 部署方案
+
+本文档介绍如何部署 Kafka 版本的 TiDB-Binlog。
+
+## TiDB-Binlog 简介
+
+TiDB-Binlog 用于收集 TiDB 的 Binlog，并提供实时备份和同步功能的商业工具。
+
+TiDB-Binlog 支持以下功能场景:
+
+* **数据同步**：同步 TiDB 集群数据到其他数据库
+* **实时备份和恢复**：备份 TiDB 集群数据，同时可以用于 TiDB 集群故障时恢复
+
+## TiDB-Binlog Kafka 架构
+
+首先介绍 TiDB-Binlog 的整体架构。
+
+![TiDB-Binlog 架构](../../media/tidb_binlog_kafka_architecture.png)
+
+TiDB-Binlog 集群主要分为三个组件：
+
+#### Pump
+
+Pump 是一个守护进程，在每个 TiDB 主机的后台运行。其主要功能是实时记录 TiDB 产生的 Binlog 并顺序写入 Kafka 中。
+
+#### Drainer
+
+Drainer 从 Kafka 中收集 Binlog，并按照 TiDB 中事务的提交顺序转化为指定数据库兼容的 SQL 语句，最后同步到目的数据库或者写到顺序文件。
+
+#### Kafka & ZooKeeper
+
+Kafka 集群用来存储由 Pump 写入的 Binlog 数据，并提供给 Drainer 进行读取。
+
+> **注**：local 版本将 Binlog 存储在文件中，最新版本则使用 Kafka 存储。
+
+## TiDB-Binlog 安装
+
+以下为 TiDB-Ansible 分支与 TiDB 版本的对应关系，版本选择可咨询官方 info@pingcap.com。
+
+    | TiDB-Ansible 分支 | TiDB 版本 | 备注 |
```

Comment:
please fix the format of this table.

Reply:
Fixed in https://github.com/pingcap/docs-cn/pull/1264/commits/6caec924771ab9b570de56a046f503cbecbdd89f